File: Ruby/Queries/LargestElementArray.rb
def large_element(arr , num)
    bubble_sort(arr,num)
    arr.last
end
def bubble_sort(arr,num)
    (0...num).each do |i|
        (0...(num - i - 1)).each do|j|
            if arr[j] > arr[j+1]
                temp = arr[j+1]
                arr[j+1] = arr[j]
                arr[j] = temp
            end
        end
    end
end
puts "no of elements"
element = gets.to_i
arr = []
(0...element).each do|i|
    arr[i] = gets.to_i
end
puts "Largest element of array is #{large_element(arr, element)}"

File: Ruby/Queries/LargestElementArray.c
#include <stdio.h>
int large_element(int num[50]){
    
}
void bubble_sort(int arr[], int num){
    for(int i = 0; i<num; i++){
        for(int j = 0; j<num - 1; j++){
            if(arr[j] > arr[j+1]){
                int temp = arr[j+1];
                arr[j+1] = arr[j];
                arr[j] = temp;
            }
        }
    }
}
int main(){
    printf("no of elements \n");
    int element;
    scanf("%d", & element);
    int arr[element];
    for(int i = 0; i<element; i++){
        scanf("%d", & arr[i]);
    }
    bubble_sort(arr, element);
    // for (int i = 0; i < element; i++){
    //     printf(" %d element is %d \n", i+1, arr[i]);
    // }
    printf("Largest element of array is %d",arr[element-1]);
    return 0;
}